proto/messages.go

```diff
@@ -1122,14 +1122,23 @@ type ConsumerMetadataResp struct {
 	CoordinatorPort int32
 }
 
-func ReadConsumerMetadataResp(r io.Reader) (*ConsumerMetadataResp, error) {
+func ReadConsumerMetadataResp(r io.Reader, version int16) (*ConsumerMetadataResp, error) {
```

Comment:
How important would it be to not break this API? If so, could we add a `ReadVersionedConsumerMetadataResp` instead?

Reply:
Not important at all.  Client never uses this function directly - only through connection methods.
But I don't understand why would  we want to have a Versioned prefix?  It would make sense if we had both Versioned and Not versioned functions, but since we all responses are versioned it doesn't make sense to emphasize it in every function name.

Reply:
The prefix would be only so that we wouldn't break consumers of this library, if you deemed important.

Reply:
I've addressed this issue in the refactoring PR. 
https://github.com/optiopay/kafka/pull/95/commits/22c80966da8c1a77f01d5ff3ce51ce7281b1907a